selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/ConnectionsManagement.java

```diff
@@ -156,12 +161,13 @@ public void acceptAConnection(String username, String... fullName) {
    * @param accept (true: if user accept invitation)
    */
   public void verifyConnection(String username, Boolean accept) {
-    evt.click(ELEMENT_MY_CONNECTIONS_TAB);
+
+    $(ELEMENT_MY_CONNECTIONS_TAB).click();
     // With user confirmed the invitation, user becomes friend and user's name
     // is displayed on user's network list
     searchPeople(username, null, null, null);
     if (accept)
-      evt.waitForAndGetElement(ELEMENT_CONNECTION_REVOVE_BTN.replace("${user}", username));
+    $(byText("Remove Connection")).should(Condition.exist);
```

Comment:
Please use a variable in SocialLocator instead of using string, to be compliant with other tests.